src/doom/g_game.c

```diff
@@ -2127,6 +2132,10 @@ static const char *DemoVersionDescription(int version)
 
     switch (version)
     {
+        case 101:
+            return "v1.0/v1.1";
+        case 102:
```

Comment:
Same here. We can't detect this.

Reply:
Reverted in https://github.com/chocolate-doom/chocolate-doom/pull/1205/commits/77f49e5ea5aae45fe904306a9364018c0fc23b71 .